src/ChannelManager.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <unordered_map>
+#include <set>
+
+#include "StringUtils.h"
+
+
+class cClientHandle;
+class cCriticalSection;
+class cPluginLua;
+class cWorld;
+class cChannelCallback;
+
+
+// tolua_begin
+class cChannelManager
+{
+// tolua_end
+public:
+	typedef SharedPtr<cChannelCallback> cChannelCallbackPtr;
+	typedef std::unordered_map<AString, cChannelCallbackPtr> CallbackMap;
+
+	void AddClientToChannel      (cClientHandle & a_Client, const AString & a_Channel);
+	void AddClientToChannels     (cClientHandle & a_Client, const AString & a_Channels);
+	void RemoveClientFromChannel (cClientHandle & a_Client, const AString & a_Channel);
+	void RemoveClientFromChannels(cClientHandle & a_Client, const AString & a_Channels);
```

Comment:
Add doxycomments for the format of `a_Channels` - a string of NUL-separated names.